Documentation/ApiOverview/Database/QueryBuilder/Index.rst

```diff
@@ -847,6 +847,84 @@ Remarks:
 *   For more complex statements you can use the raw Doctrine QueryBuilder.
     See remarks for :ref:`orderBy() <database-query-builder-orderby>`
 
+..  _database-query-builder-union:
+
+union() and addUnion()
+======================
+
+Method `union()` provides a streamlined way to combine result sets from multiple
+queries.
+
+:php:`union(string|QueryBuilder $part)`
+    Creates the initial :sql:`UNION` query part by accepting either a raw SQL
+    string or a `QueryBuilder` instance.
```

Comment:
Maybe we can somehow emphasize that calling `union()` will reset any union parts added with `union()` and subsequential `addUnion()` ?

Reply:
Done